### PR TITLE
mechanism for send email

### PR DESCRIPTION
**Pull request by ps-eqsdigital**
utils for email sender

### CODE REVIEW

File: api/Business/Utils/EmailUtils.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                client.Credentials = new NetworkCredential(smtpUsername, smtpPassword);
+
+                MailMessage message = new MailMessage();
+                message.From = new MailAddress(smtpUsername);
+                message.To.Add(email);
+                message.Subject = subject;
+                message.Body = content;
+
+                client.Send(message);
+                Console.WriteLine("Email sent successfully.");
```
Comment (tentone) [inline api/Business/Utils/EmailUtils.cs]: Remove prints

File: api/Business/Utils/EmailUtils.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json")
+                .Build();
+
+                var smtpServer = configuration["SmtpConfig:SmtpServer"];
+                var smtpPortString = configuration["SmtpConfig:SmtpPort"];
+                var smtpUsername = configuration["SmtpConfig:SmtpUsername"];
+                var smtpPassword = configuration["SmtpConfig:SmtpPassword"];
+                int smtpPort = int.Parse(smtpPortString);
+
```
Comment (tentone) [inline api/Business/Utils/EmailUtils.cs]: Declare types for everything instead of "var"

**Review by tentone — APPROVED** [review]

File: api/Business/Utils/EmailUtils.cs (hunk 1 above)
Comment (ps-eqsdigital) [reply api/Business/Utils/EmailUtils.cs]: done

File: api/Business/Utils/EmailUtils.cs (hunk 2 above)
Comment (ps-eqsdigital) [reply api/Business/Utils/EmailUtils.cs]: done

**Review by kripton69 — APPROVED** [review]

**Pull request merged**